src/lib.rs

```diff
@@ -28,6 +28,8 @@ pub struct HD44780<D: DelayUs<u16> + DelayMs<u8>, B: DataBus> {
     display_mode: DisplayMode,
 }
 
+pub struct Error;
```

Comment:
Instead of re-defining this we should re-import it from elsewhere, like the crate root?